File: 백준/Bronze/10768. 특별한 날/특별한 날.java
import java.io.*;
import java.util.*;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int month = Integer.parseInt(br.readLine());
        int day = Integer.parseInt(br.readLine());
        if (month < 2 || (month == 2 && day < 18)) {
            bw.write("Before");
        } else if (month == 2 && day == 18) {
            bw.write("Special");
        } else if ((month == 2 && day > 18) || month > 2) {
            bw.write("After");
        }

        bw.flush();
        bw.close();
        br.close();
    }
}

File: 백준/Bronze/10768. 특별한 날/특별한 날.py
month = int(input())
day = int(input())

if month == 1:
    print("Before")
elif month == 2 and day < 18:
    print("Before")
elif month == 2 and day == 18:
    print("Special")
elif month == 2 and day > 18:
    print("After")
elif month >= 3:
    print("After")
